### PR TITLE
activerecord: Allow to use array of Hashes to #preload

### PR DESCRIPTION
**Pull request by tk0miya**
The real use case is here: https://github.com/kaigionrails/conference-app/pull/341/files#r1817211252

### CODE REVIEW

File: gems/activerecord/6.0/activerecord.rbs [hunk 1]
```diff
@@ -296,6 +296,8 @@ module ActiveRecord
   end
 
   module QueryMethods
+    type includee = String | Symbol | Array[includee] | Hash[untyped, untyped]
```
Comment (tk0miya) [inline gems/activerecord/6.0/activerecord.rbs:299]: I'm not sure this is the best place for this type alias. I'd really appreciate if you give me your opinion.
Thanks,

Comment (tk0miya) [reply gems/activerecord/6.0/activerecord.rbs:299]: I don't have a strong opinion of accuracy. So I'm okay to stop using recurrent definitions like this.

```suggestion
    type includee = String | Symbol | Array[untyped] | Hash[untyped, untyped]
```

Comment (tk0miya) [reply gems/activerecord/6.0/activerecord.rbs]: Finally, I removed `includee` type alias. It's only needed for recurrent definitions.

**Review by ksss — COMMENTED** [review]
APPROVE

**Pull request merged**